packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/default.parser.spec.ts

```diff
@@ -1,3 +1,135 @@
-describe("default Parser", () => {
-  it("TODO - add tests", () => expect(true).toEqual(true));
+import { FlowTypes } from "data-models";
+import { DefaultParser } from "./default.parser";
+
+const testInputSources = {
+  data_list: { test_data_list: [{ id: 1 }, { id: 2 }, { id: 3 }] },
+};
+
+describe("Default Parser", () => {
+  it("Cleans field values - removes whitespace within empty strings", () => {
+    const parser = new DefaultParser({ processedFlowHashmap: testInputSources } as any);
+    const output = parser.run({
+      flow_name: "test_flow_input_empty",
+      flow_subtype: "generator",
+      flow_type: "generated",
+      rows: [
+        {
+          type: "text",
+          name: "empty",
+          value: "",
+        },
+        {
+          type: "text",
+          name: "tab_empty",
+          value: "\t\t",
+        },
+        {
+          type: "text",
+          name: "newline_empty",
+          value: "\n\n",
+        },
+        {
+          type: "text",
+          name: "return_empty",
+          value: "\r\r",
+        },
+        {
+          type: "text",
+          name: "newline_return_empty",
+          value: "\r\n\n\r",
+        },
+      ],
+    }) as FlowTypes.FlowTypeWithData;
+    // not possible to use the actual objects as the function used is not exported
+    expect(output.rows).toEqual([
+      {
+        type: "text",
+        name: "empty",
+        value: "",
+      },
+      {
+        type: "text",
+        name: "tab_empty",
+        value: "",
+      },
+      {
+        type: "text",
+        name: "newline_empty",
+        value: "",
+      },
+      {
+        type: "text",
+        name: "return_empty",
+        value: "",
+      },
+      {
+        type: "text",
+        name: "newline_return_empty",
+        value: "",
+      },
+    ]);
+  });
+
+  it("Cleans field values - Replaces string consisting of only line breaks and new lines", () => {
```

Comment:
This description isn't representative of what the spec test is checking. "Cleans field values - trims whitespace from beginning and end of strings"